### ISSUE TITLE
the example model of 'SGPT-125M-weightedmean-nli-bitfit' not compatible with ST

### ISSUE THREAD

**Issue by dengwx2009** [Muennighoff/sgpt#29]
hi, I tried to run the example be_st with 'Muennighoff/SGPT-125M-weightedmean-nli-bitfit'
but it throw out the exception as follows:
`
/usr/local/lib/python3.8/site-packages/sentence_transformers/models/Pooling.py:120 in load
TypeError: __init__() got an unexpected keyword argument 'pooling_mode_weightedmean_tokens'
`
Is ’pooling_mode_weightedmean_tokens‘ in the config.json changed to ’pooling_mode_mean_tokens’ ？

**Comment by Muennighoff** [comment]
Can you try what is explained here: https://github.com/Muennighoff/sgpt/issues/14#issuecomment-1352628910
You're likely using an old ST version

**Comment by MattiL** [comment]
I got the same error with the computing of the web interface at https://huggingface.co/bigscience-data/sgpt-bloom-1b7-nli a moment ago.

**Comment by Muennighoff** [comment]
> I got the same error with the computing of the web interface at https://huggingface.co/bigscience-data/sgpt-bloom-1b7-nli a moment ago.

Yes that's because the web interface is not using the latest ST version
